data/xml/1995.iwpt.xml

```diff
@@ -0,0 +1,287 @@
+      <editor><first>Harry</first><last>Bunt</last></editor>
+      <editor><first>Bob</first><last>Carpenter</last></editor>
+      <editor><first>Ken</first><last>Church</last></editor>
+      <editor><first>Aravind</first><last>Joshi</last></editor>
+      <editor><first>Ronald</first><last>Kaplan</last></editor>
+      <editor><first>Martin</first><last>Kay</last></editor>
+      <editor><first>Makoto</first><last>Nagao</last></editor>
+      <editor><first>Anton</first><last>Nijholt</last></editor>
+      <editor><first>Mark</first><last>Steedman</last></editor>
+      <editor><first>Henry</first><last>Thompson</last></editor>
+      <editor><first>Masaru</first><last>Tomita</last></editor>
+      <editor><first>K.</first><last>Vijay-Shanker</last></editor>
+      <editor><first>Yorick</first><last>Wilks</last></editor>
+      <editor><first>Kent</first><last>Wittenburg</last></editor>
+    </meta>
+    <paper id="1">
+      <title>Proceedings of the Fourth International Workshop on Parsing Technologies</title>
+      <pages>i-ix</pages>
+      <url>1995.iwpt-1.1</url>
+      <abstract></abstract>
+    </paper>
+    <paper id="2">
+      <title>Acyclic Context-sensitive Grammars</title>
+      <author><first>Erik</first><last>Aarts</last></author>
+      <pages>1-9</pages>
+      <url>1995.iwpt-1.2</url>
+      <abstract>A grammar formalism is introduced that generates parse trees with crossing branches. The uniform recognition problem is NP-complete, but for any fixed grammar the recognition problem is polynomial.</abstract>
+    </paper>
+    <paper id="3">
+      <title>PARSING IN DIALOGUE SYSTEMS USING TYPED FEATURE STRUCTURES</title>
+      <author><first>Rieks</first><last>op den Akker</last></author>
+      <author><first>Hugo</first><last>ter Doest</last></author>
+      <author><first>Mark</first><last>Moll</last></author>
+      <author><first>Anton</first><last>Nijholt</last></author>
+      <pages>10-11</pages>
+      <url>1995.iwpt-1.3</url>
+      <abstract>The analysis of natural language in the context of keyboard-driven dialogue systems is the central issue addressed in this paper. A module that corrects typing errors, performs domain-specific morphological analysis is developed. A parser for typed unification grammars is designed and implemented in C++; for description of the lexicon and the grammer a specialised specification language is developed. It is argued that typed unification grammars and especially the newly developed specification language are convenient formalisms for describing natural language use in dialogue systems. Research on these issues is carried out in the context of the SCHISMA project, a research project in linguistic engineering; participants in SCHISMA are KPN Research and the University of Twente.</abstract>
+    </paper>
+    <paper id="4">
+      <title>Parallel Parsing: Different Distribution Schemata for Charts</title>
+      <author><first>Jan W.</first><last>Amtrup</last></author>
```

Comment:
probably should make this block caps for consistency with the rest of the volume